vignettes/Vaccines.Rmd

```diff
@@ -40,153 +123,262 @@ simparams <- get_parameters(list(
 
 simparams <- set_equilibrium(simparams, starting_EIR)
 
+# Run a model with no interventions in a setting with no seasonality
+output_control <- run_simulation(sim_length * 2, simparams)
 ```
 
-Then we can run the simulation for a variety of Vaccination strategies:
+Then we can run the simulation for a variety of vaccination strategies.
 
 ## Mass RTS,S
 
-This is a round of RTS,S vaccine for individuals between 5 months and 10 years followed by a booster after 18 months:
+First, we will model a single round of RTS,S vaccine given in a mass vaccination strategy where those within a given age range will be vaccinated to a specified coverage level and during a short window of time. In this example, individuals aged between 5 months and 10 years are vaccinated with a primary series (3 doses) followed by a single booster 12 months after the initial vaccination. The age groups to be vaccinated as well as timing of all vaccinations can be modified within the `set_mass_rtss()` function.
```

Comment:
let's change to a realistic time window with the literature so far instead of up to 10 years, esp since your plotted outputs are for ages 0-5 for incidence